### ISSUE TITLE
Loading Key Backup File Crash

### ISSUE THREAD

**Issue by HashEngineering** [HashEngineering/dash-wallet#107]
Loading a backup from over 3 years ago will crash the app.  These backups do not have keychains.

```
org.bitcoinj.wallet.DeterministicUpgradeRequiredException
at org.bitcoinj.wallet.KeyChainGroup.getActiveKeyChain(KeyChainGroup.java:256)
at org.bitcoinj.wallet.Wallet.getKeyChainSeed(Wallet.java:1086)
at org.bitcoinj.wallet.Wallet.addKeyChain(Wallet.java:5381)
at de.schildbach.wallet.ui.WalletActivity.upgradeWalletKeyChains(WalletActivity.java:1101)
at de.schildbach.wallet.ui.WalletActivity.restoreWallet(WalletActivity.java:970)
at de.schildbach.wallet.ui.WalletActivity.restoreWalletFromEncrypted(WalletActivity.java:883)
at de.schildbach.wallet.ui.WalletActivity.access$500(WalletActivity.java:105)
at de.schildbach.wallet.ui.WalletActivity$9.onClick(WalletActivity.java:553)
at com.android.internal.app.AlertController$ButtonHandler.handleMessage(AlertController.java:163)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5417)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Comment by HashEngineering** [comment]
No google or user reports of this issue.

After this type of backup is restored, check for the existance any keychain, if not create them.  Or determine when the app would create the keychains and put the upgrade code there.

**Comment by HashEngineering** [comment]
Fixed with 623f45fcecc7c393bc74d5e1205ba76ae439bc55 for key files with no PIN.
